po/de.po

```diff
@@ -7147,10 +7142,10 @@ msgid ""
 "any valid strftime string; for more information about this format, read the "
 "manual page of strftime by \"man 3 strftime\"."
 msgstr ""
-"Wenn das Datumsformat auf Benutzerdefiniert gesetzt ist, kann in dieser Ausdruck"
-"als Argument an das strftime-Format übergegeben werden. Es kann eine beliebige "
-"gültige strftime Zeichenkette sein. Für mehr Informationen "
-"lesen Sie die Ausgabe von »man 3 strftime«."
+"Wenn das Datumsformat auf Benutzerdefiniert gesetzt ist, kann in dieser "
+"Ausdruckals Argument an das strftime-Format übergegeben werden. Es kann eine "
```

Comment:
This looks like an error to me: I think it should stay "Ausdruck als Argument" and not become "Ausdrukals Argument". But that may very well have been an error in the original translation as well. I think that first line should have ended in a space.